WinImageBuilder.psm1

```diff
@@ -543,6 +543,20 @@ function Add-DriversToImage {
     }
 }
 
+function Add-PackageToImage {
+    Param(
+        [Parameter(Mandatory=$true)]
+        [string]$winImagePath,
+        [Parameter(Mandatory=$true)]
+        [string]$packagePath
+    )
+    Write-Host ('Adding packages from "{0}" to image "{1}"' -f $packagePath, $winImagePath)

```

Comment:
@ader1990,

I've updated the commit like you suggested. However, after reviewing the other status messages in the file, it looks like `Write-Log` is used most everywhere else.

`Write-Log` calls `Write-Host` ... should I use that instead?

Reply:
Yes, Write-Log would be great as it has also a timestamp